require.js

```diff
@@ -1119,12 +1149,46 @@
     };
 
 
+    Require.parseMJSONDependencies = function parseMJSONDependencies(jsonRoot) {
```

Comment:
This function doesn't need a name, it's only called via `Require.parseMJSONDependencies`.

Reply:
It does't, but it helps a lot for tasks like memory optimizations using developer tools

Reply:
Ok